### ISSUE TITLE
export directly from Obsidian

### ISSUE THREAD

**Issue by alixander** [terrastruct/d2-obsidian#8]
https://discord.com/channels/1039184639652265985/1039185315685998693/1050768272431271937

**Comment by liefra** [comment]
Is this already documented somewhere? Cannot see it in version 1.1.3. (Just bought the TALA license yesterday to use with with Obsidian)

**Comment by alixander** [comment]
good point. @berniexie can you document this please? 

I believe in this case it's just Obsidian's default export. e.g. File > Export PDF

**Comment by liefra** [comment]
Yes, the File -> Export to PDF works. 

But after reviewing the Discord thread I had the impression that this functionality is addressing a direct SVG/PNG export. 

Is it PDF export only? (would be not that useful for me)

**Comment by alixander** [comment]
@liefra yes I believe so. I don't think Obsidian allows for that type of direct export. There's nothing we can do there. You'd have to use the CLI for that.

**Comment by liefra** [comment]
If there is no direct Obsidian way for SVG/PNG export, maybe the plugin could provide a clipboard-able command which does a SVG/PNG export for the given Obsidian note or a specific D2 diagram inside a note? 

Not sure if it would be helpful for others?

**Comment by alixander** [comment]
will revisit based on @nlundquist 's comment.

note to look at draw.io's svg export
